support/cas-server-support-consent-couchdb/src/main/java/org/apereo/cas/consent/CouchDbConsentRepository.java

```diff
@@ -0,0 +1,83 @@
+package org.apereo.cas.consent;
+
+import org.apereo.cas.authentication.Authentication;
+import org.apereo.cas.authentication.principal.Service;
+import org.apereo.cas.couchdb.ConsentDecisionCouchDbRepository;
+import org.apereo.cas.couchdb.CouchDbConsentDecision;
+import org.apereo.cas.services.RegisteredService;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.ektorp.DbAccessException;
+
+import java.util.Collection;
+import java.util.stream.Collectors;
+
+/**
+ * This is {@link CouchDbConsentRepository}.
+ *
+ * @author Timur Duehr
+ * @since 6.0.0
+ */
+@Getter
+@Setter
+@AllArgsConstructor
```

Comment:
RequiredArgsConstructor would be better, if you make fields be final.

Reply:
Done